src/app/blog/index.html

```diff
@@ -4,6 +4,18 @@
 	windowTitle="SKY UX Blog"
 	showTableOfContents="true">
 	
+	<article class="blog-post">
+		<header>
+			<stache-page-anchor>
+				Design update — June 2020
+			</stache-page-anchor>
+		</header>
+		<section>
+			<h3 class="sky-subsection-heading">Recently completed</h3>
+			<p>We are in the process of cleaning up some accessibility gaps.</p>
```

Comment:
"filling in" instead of "cleaning up"? or maybe "We are addressing accessibility issues." ?